modules/core_api/fsw/inc/cfe_msg_api_typedefs.h

```diff
@@ -45,7 +45,7 @@
 /*
  * Types
  */
-typedef size_t CFE_MSG_Size_t;          /**< \brief Message size (CCSDS needs uint32 for max size) */
+typedef size_t CFE_MSG_Size_t;          /**< \brief Message size, note CCSDS maximum is MAX_UINT16+7 */
```

Comment:
Minor nitpick: C99-defined limit value is `UINT16_MAX`, not `MAX_UINT16`

Reply:
Fixed in e4e3456